### DISCUSSION TITLE
Link to RKI: COVID-19 Dashboard (DE)

### DISCUSSION THREAD

**Discussion by MikeMcC399** [corona-warn-app/cwa-website#3533] [Q&A] [answered]
[RKI: COVID-19 Dashboard (DE)](https://experience.arcgis.com/experience/478220a4c454480e823b17327b2bf1d4)

now shows

![image](https://github.com/corona-warn-app/cwa-website/assets/66998419/1059577c-d195-4c30-a4b3-c8926259a5c7)

so probably it can be removed from https://www.coronawarn.app/en/analysis/

**Comment by MikeMcC399** [comment]
June 3 and June 4, 2023 showing:

![RKI dashboard 2023-06-03](https://github.com/corona-warn-app/cwa-website/assets/66998419/89387698-bb35-422a-a16f-ad76fff91e82)

**Reply by MikeMcC399** [reply]
Maybe it's best to leave the link and rely on https://experience.arcgis.com/experience/478220a4c454480e823b17327b2bf1d4 to provide the correct information about the state of the site?

**Comment by MikeMcC399** [comment]
Apparently the server function has now been deactivated. There was a notice at the weekend that this would happen in calendar week 23/2023, which began today June 5, 2023. So in fact, it probably **does** make sense to remove the link.

![image](https://github.com/corona-warn-app/cwa-website/assets/66998419/49c81148-8fe0-4050-9050-52ee3b337343)

**Comment by dsarkar** [comment, marked as answer]
@MikeMcC399 Thanks for the hint. See https://github.com/corona-warn-app/cwa-website/pull/3535
